src/components/CountDownTimer.tsx

```diff
@@ -0,0 +1,108 @@
+import { CountdownCircleTimer } from "react-countdown-circle-timer";
+import "../styles.css";
+
+const minuteSeconds = 60;
+const hourSeconds = 3600;
+const daySeconds = 86400;
```

Comment:
since these are constants, they should be in screaming snake case. e.g. SECONDS_IN_MINUTE = 60; SECONDS_IN_HOUR = SECONDS_IN_MINUTE * 60;, etc